diff --git a/src/github.com/bazelbuild/bazel-gazelle/cmd/gazelle/langs.go b/src/github.com/bazelbuild/bazel-gazelle/cmd/gazelle/langs.go
index 0f8e112..ee5a180 100644
--- a/src/github.com/bazelbuild/bazel-gazelle/cmd/gazelle/langs.go
+++ b/src/github.com/bazelbuild/bazel-gazelle/cmd/gazelle/langs.go
@@ -19,9 +19,11 @@ import (
 	"github.com/bazelbuild/bazel-gazelle/language"
 	"github.com/bazelbuild/bazel-gazelle/language/go"
 	"github.com/bazelbuild/bazel-gazelle/language/proto"
+	"github.com/iocat/rules_gleam/gazelle/gleam"
 )
 
 var languages = []language.Language{
+	gleam.NewLanguage(),
 	proto.NewLanguage(),
 	golang.NewLanguage(),
 }
